integration/inspektor-gadget/advise_networkpolicy_test.go

```diff
@@ -41,25 +47,92 @@ func TestAdviseNetworkpolicy(t *testing.T) {
 		WaitUntilTestPodReadyCommand(nsClient),
 		{
 			Name: "RunNetworkPolicyMonitorClient",
-			Cmd: fmt.Sprintf(`$KUBECTL_GADGET advise network-policy monitor -n %s --output ./networktrace-client.log &
-					sleep 10
-					kill $!
-					echo
-					grep tcp networktrace-client.log | sort | uniq`, nsClient),
-			ExpectedRegexp: fmt.Sprintf(`{"node":".*","namespace":"%s","pod":"test-pod",.*"type":"normal",.*"pktType":"OUTGOING","proto":"tcp","port":9090,.*"podHostIP":".*","podIP":".*","podLabels":{"run":"test-pod"},"remoteKind":"svc","remoteAddr":".*","remoteName":"test-pod","remoteNamespace":"%s","remoteLabels":{"run":"test-pod"}}`, nsClient, nsServer),
+			Cmd:  fmt.Sprintf(`$KUBECTL_GADGET advise network-policy monitor -n %s --timeout 5 --output - | tee ./networktrace-client.log`, nsClient),
+			ExpectedOutputFn: func(output string) error {
+				expectedEntry := &networkTypes.Event{
+					Event:           BuildBaseEvent(nsClient),
+					Comm:            "nc",
+					Uid:             0,
+					Gid:             0,
+					PktType:         "OUTGOING",
+					Proto:           "tcp",
+					PodLabels:       map[string]string{"run": "test-pod"},
+					Port:            9090,
+					RemoteKind:      eventtypes.RemoteKindService,
+					RemoteNamespace: nsServer,
+					RemoteName:      "test-pod",
+					RemoteLabels:    map[string]string{"run": "test-pod"},
+				}
+
+				expectedEntry.Container = ""
```

Comment:
Note: the "trace network" gadget sets the field `Container` via enrichers, but "advise network-policy" does not because it does not use the operators yet (it still uses the CR path). We can remove this line from the test once the gadget gets converted.